sheep/sheep.c

```diff
@@ -1028,13 +1028,12 @@ int main(int argc, char **argv)
 		goto cleanup_journal;
 
 	#ifdef HAVE_HTTP
-	if (http_options && http_init(http_options) != 0)
```

Comment:
Removing the variable is overkill, of course. Just excluding its declaration into ifdef is ok.